common/core/web/utils/package.json

```diff
@@ -5,7 +5,7 @@
   "main": "./dist/index.js",
   "types:": "./dist/index.d.ts",
   "scripts": {
-    "build": "tsc -p ./src/tsconfig.json",
+    "build": "cd src/ && ./build.sh -skip-dependency-install",
```

Comment:
Not sure that this will run correctly on Windows because it uses shisms. cmd is such a horrible command processor but npm doesn't make our lives easier by the way it embeds platform-specific scripts...

I ended up using `run-script-os` npm package to do platform-specific scripts on status.keyman.com. Escaping of parameters is a little fraught but it got things working (had to wrap them in a script to get them going). See https://github.com/keymanapp/status.keyman.com/pull/45/files

Reply:
A possible icky alternative is that we enforce bash as our script shell on Windows for NPM. But that's a tricky setting because we don't want it to be enforced globally as that will break non-Keyman NPM configurations -- yet it is machine dependent so we can't really set it in the repo-level settings!

https://stackoverflow.com/a/46006249/1836776

We could use environment variables to control this at a build level but far out more configuration.